BU-Spark/se-bch-als-resource-app
# updated current progress

- fixed acccess to strapi admin panel
- home page buttons are set up
- routing to subpage(no relation entry on strapi see below)
- customized script for strapi collection input(work in progress)

issues:
- Import-script.js: 405 Method NOt Allowed AxiosError: Request failed with status code 405 

Possible sources: 
- Strapi might require authentitcaiton for API request, strapi's RBAC resttrict access to API endpoints that is used in the script

To do/tests:
- Check Strapi Authentication Endpoint, testing API endpoint : http://localhost:1338/auth/local
- review strapi configuration on public & authenticated roles
- update STRAPI_URL in import-script.js(change IP address)
